Formula/acmetool.rb

```diff
@@ -17,165 +16,22 @@ class Acmetool < Formula
-    url "https://github.com/ogier/pflag.git",
-        :revision => "45c278ab3607870051a2ea9040bb85fcb8557481"
-  end
-
-  go_resource "github.com/peterhellberg/link" do
-    url "https://github.com/peterhellberg/link.git",
-        :revision => "8768c6d4dc563b4a09f58ecda04997024452c057"
-  end
-
-  go_resource "github.com/satori/go.uuid" do
-    url "https://github.com/satori/go.uuid.git",
-        :revision => "36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
-  end
-
-  go_resource "github.com/shiena/ansicolor" do
-    url "https://github.com/shiena/ansicolor.git",
-        :revision => "a422bbe96644373c5753384a59d678f7d261ff10"
-  end
-
-  go_resource "golang.org/x/crypto" do
-    url "https://go.googlesource.com/crypto.git",
-        :revision => "a6600008915114d9c087fad9f03d75087b1a74df"
-  end
-
-  go_resource "golang.org/x/net" do
-    url "https://go.googlesource.com/net.git",
-        :revision => "5ccada7d0a7ba9aeb5d3aca8d3501b4c2a509fec"
-  end
-
-  go_resource "golang.org/x/sys" do
-    url "https://go.googlesource.com/sys.git",
-        :revision => "2c42eef0765b9837fbdab12011af7830f55f88f0"
-  end
-
-  go_resource "golang.org/x/text" do
-    url "https://go.googlesource.com/text.git",
-        :revision => "e19ae1496984b1c655b8044a65c0300a3c878dd3"
-  end
-
-  go_resource "gopkg.in/alecthomas/kingpin.v2" do
-    url "https://gopkg.in/alecthomas/kingpin.v2.git",
-        :revision => "947dcec5ba9c011838740e680966fd7087a71d0d"
-  end
-
-  go_resource "gopkg.in/cheggaaa/pb.v1" do
-    url "https://gopkg.in/cheggaaa/pb.v1.git",
-        :revision => "43d64de27312b32812ca7e994fa0bb03ccf08fdf"
-  end
-
-  go_resource "gopkg.in/hlandau/configurable.v1" do
-    url "https://gopkg.in/hlandau/configurable.v1.git",
-        :revision => "41496864a1fe3e0fef2973f22372b755d2897402"
-  end
-
-  go_resource "gopkg.in/hlandau/easyconfig.v1" do
-    url "https://gopkg.in/hlandau/easyconfig.v1.git",
-        :revision => "7589cb96edce2f94f8c1e6eb261f8c2b06220fe7"
-  end
-
-  go_resource "gopkg.in/hlandau/service.v2" do
-    url "https://gopkg.in/hlandau/service.v2.git",
-        :revision => "b64b3467ebd16f64faec1640c25e318efc0c0d7b"
-  end
-
-  go_resource "gopkg.in/hlandau/svcutils.v1" do
-    url "https://gopkg.in/hlandau/svcutils.v1.git",
-        :revision => "c25dac49e50cbbcbef8c81b089f56156f4067729"
-  end
-
-  go_resource "gopkg.in/square/go-jose.v1" do
-    url "https://gopkg.in/square/go-jose.v1.git",
-        :revision => "aa2e30fdd1fe9dd3394119af66451ae790d50e0d"
-  end
-
-  go_resource "gopkg.in/tylerb/graceful.v1" do
-    url "https://gopkg.in/tylerb/graceful.v1.git",
-        :revision => "4654dfbb6ad53cb5e27f37d99b02e16c1872fbbb"
-  end
-
-  go_resource "gopkg.in/yaml.v2" do
-    url "https://gopkg.in/yaml.v2.git",
-        :revision => "d670f9405373e636a5a2765eea47fac0c9bc91a4"
+  patch do
+    url "https://github.com/CL-Jeremy/acmetool/commit/7b7ac8c75e936dcefff2c8b2903eb5c442db345f.patch?full_index=1"
```

Comment:
```suggestion
    url "https://github.com/hlandau/acmetool/commit/7b7ac8c75e936dcefff2c8b2903eb5c442db345f.patch?full_index=1"
```

Reply:
I searched a bit in the repo and did not find much about this case. I think the concern is I may delete this fork afterwards, right? Then it makes sense, but I've not yet planned to make a PR to upstream about this. Is it okay to do so?

Reply:
Ah, in that case just open a pull request with the patch to [Homebrew/formula-patches](https://github.com/Homebrew/formula-patches) and update the URL accordingly. Yes, the concern is that the patch will vanish.